.github/workflows/android.yml

```diff
@@ -0,0 +1,28 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Do you need this, or just the "pull_request"?

Reply:
Discussed over Zoom.  We will include both.

Reply:
https://github.com/simplymadeapps/PreferenceHelper/pull/12/commits/dca3dbc9dec5136d674e6a7f284d10e28d2ab5e3